app/services/vot/parser.rb

```diff
@@ -4,37 +4,71 @@ module Vot
   class Parser
     class ParseException < StandardError; end
 
-    Result = Data.define(
+    class Result < Data.define(
+      :type,
```

Comment:
why did this `type` value get added?

Reply:
I had a thought about the reporting on this, but it doesn't seem needed, so will remove.